app/grandchallenge/reader_studies/serializers.py

```diff
@@ -73,6 +74,17 @@ class DisplaySetSerializer(HyperlinkedModelSerializer):
     view_content = JSONField(
         source="reader_study.view_content", read_only=True
     )
+    description = SerializerMethodField()
+
+    def get_description(self, obj):
```

Comment:
Could you move this to a `description` `@property` on the model? Then add it to each display set in the display set list view? Would make it easier for editors to track.

This doesn't introduce any N+1 problems?

Reply:
Also would make testing easier as you could then just do a model test, rather than going through the API.

Reply:
Done, and now that I have actually added the prefetch_related to the view it does not.